As resquested: change to be aligned with the vendor KB

From: Yann LE BRIS <yann.lebris@vates.tech>

Signed-off-by: Yann LE BRIS <yann.lebris@vates.tech>

Index: sm-3.2.12/udev/99-purestorage.rules
===================================================================
--- sm-3.2.12.orig/udev/99-purestorage.rules
+++ sm-3.2.12/udev/99-purestorage.rules
@@ -12,4 +12,4 @@ ACTION=="add|change", KERNEL=="sd*[!0-9]
 ACTION=="add|change", KERNEL=="dm-[0-9]*", SUBSYSTEM=="block", ENV{DM_NAME}=="3624a937*", ATTR{queue/rq_affinity}="2"
 
 # Set the HBA timeout to 60 seconds
-ACTION=="add|change", KERNEL=="sd*[!0-9]", SUBSYSTEM=="block", ENV{ID_VENDOR}=="PURE", ATTR{device/timeout}="60"
\ No newline at end of file
+ACTION=="add", SUBSYSTEMS=="scsi", ATTRS{model}=="FlashArray      ", RUN+="/bin/sh -c 'echo 60 > /sys/$DEVPATH/device/timeout'"
